app/abilities/sac_cas/event/participation_ability.rb

```diff
@@ -25,4 +26,8 @@ module SacCas::Event::ParticipationAbility
   def unless_her_own
     !her_own
   end
+
+  def if_can_edit_course_or_her_own
+    her_own || Ability.new(user).can?(:edit, participation.event)
```

Comment:
Das Ability instantieren ist eher ein antipattern und sollten wir nur in Ausnahmefällen machen, kriegen wir das nicht anders hin, sollte nicht sowas auch reichen, dann könnte auch der check aus dem partial weg

```
def if_self_employed_and_her_own_or_for_participations_full_events
  partiicpation.role.self_employed? && participation.self_her_own_or_for_participations_full_events
end
```